tiny_hippo_run.py

```diff
@@ -93,6 +98,27 @@ def packet_parse(packet: Packet):
         run_config.log_event.info('Exception raised: ' + str(e))
 
 
+def __pair_ip_to_mac():
+    """
+    Uses the "arp" system call to pair IoT MAC addrs to their current IP addrs
+    :return: dict of IPs to MACs
+    """
+    # Get ARP data from the system and parse out the IP and MAC addrs
+    with open("/proc/net/arp", "r") as file:
+        arp_data = file.read()
+    arp_lines = arp_data.split("\n")
+    ip_to_mac_all = {}
+    for line in arp_lines[1:-1]:
+        line_parsed = list(filter(None, line.split(" ")))
+        ip_to_mac_all[line_parsed[0]] = line_parsed[3]
+
+    # Filter out non-IoT devices from ip_to_mac_all
+    ip_to_mac_iot = {}
+    for ip in ip_to_mac_all.keys():
+        if ip_to_mac_all[ip] in mac_addrs:
+            ip_to_mac_iot[ip] = ip_to_mac_all[ip]
+    return(ip_to_mac_iot)
+
```

Comment:
This function is fine for now I think, however once sqlite is in place we can probably rework this to first try the IPs associated with each MAC in the table, and if they are out of date to use this function.

Reply:
Alternatively we can leverage scapy instead of using a system call here: https://stackoverflow.com/questions/38182966/obtain-ip-address-from-mac-using-python